test/test_torch.py

```diff
@@ -10013,6 +10015,23 @@ def test_ops(op, string_of_function_name, expected_output):
                                                        [0, 0, 0],
                                                        [0, 0, 0]]))
 
+    @unittest.skipIf(not TEST_NUMPY, 'Numpy not found')
+    def test_cumlogsumexp(self):
+        a = torch.randn(5, 4)
+        a[0, 0] = inf
+        a[1, :] = -inf
+        actual = a.cumlogsumexp(1)
+        max = np.maximum.accumulate(a.numpy(), axis=1)
+        expected = max + np.log(np.cumsum(np.exp(a - max), axis=1))
+        self.assertEqual(a.dtype, actual.dtype)
+        self.assertEqual(expected.shape, actual.shape)
```

Comment:
Add a test to check if there's a RuntimeError when the dtype for provided output variable is not the same as input